src/veil/ringct/anonwallet.cpp

```diff
@@ -1317,12 +1318,15 @@ bool AnonWallet::ExpandTempRecipients(std::vector<CTempRecipient> &vecSend, std:
                 rd.nType = OUTPUT_DATA;
 
                 if (0 != MakeStealthData(r.sNarration, sx.prefix, sShared, pkEphem, rd.vData, r.nStealthPrefix, sError)) {
```

Comment:
Change this from `0 != `. Usual Particl weirdness.

Reply:
ahh yes, of course.  I made the decision to flip them for the ones that I was fixing their sError definitions of; which was after I went through MakeStealthData.  I made sure putting the commented out code back in will actually not break things, but I didn't go through and define sError in those places because that needs more attention anyway.

Reply:
Eventually should be a goal to revisit and rework much of the RingCT work Particl had done as it isn't Object Orientated, and has repeated code everywhere. No harm is fixing some as it comes :)